SchoolManagementSystemApi/Model/Subjects.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.ComponentModel.DataAnnotations;
```

Comment:
@New-Anointing Is there a reason why you removed the shortcode, your model is looking scanty. The property could have been called course code or something

Reply:
didn't add one before i will add that now